fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/TombstoneImpl.java

```diff
@@ -58,4 +60,19 @@ public void delete() {
             throw new RepositoryRuntimeException(e);
         }
     }
+
+    @Override
+    public String toString() {
```

Comment:
See, that's what I'm talking about. I just like that more. Couldn't tell you why.